tests/sentry/api/endpoints/test_group_user_reports.py

```diff
@@ -1,15 +1,67 @@
 from __future__ import absolute_import
-from sentry.testutils import UserReportEnvironmentTestCase
 from exam import fixture
+import logging
 
+from sentry.models import Environment, GroupStatus, UserReport
+from sentry.testutils import APITestCase, SnubaTestCase
+
+
+class GroupUserReport(APITestCase, SnubaTestCase):
+
+    def setUp(self):
+        self.project = self.create_project()
+        self.env1 = self.create_environment(self.project, 'production')
+        self.env2 = self.create_environment(self.project, 'staging')
+
+        self.group = self.create_group(project=self.project, status=GroupStatus.UNRESOLVED)
+
+        self.env1_events = self.create_events_for_environment(self.group, self.env1, 5)
+        self.env2_events = self.create_events_for_environment(self.group, self.env2, 5)
+
+        self.env1_userreports = self.create_user_report_for_events(
+            self.project, self.group, self.env1_events, self.env1)
+        self.env2_userreports = self.create_user_report_for_events(
+            self.project, self.group, self.env2_events, self.env2)
 
-class GroupUserReport(UserReportEnvironmentTestCase):
     @fixture
     def path(self):
         return u'/api/0/groups/{}/user-feedback/'.format(
             self.group.id,
         )
 
+    def create_environment(self, project, name):
+        env = Environment.objects.create(
+            project_id=project.id,
+            organization_id=project.organization_id,
+            name=name,
+        )
+        env.add_project(project)
+        return env
```

Comment:
This looks very similar to the `create_environment()` method that sentry/testutils/factories exposes. Do we need both?

Reply:
Oh good catch, can probably delete this one